spec/features/sdr_deposit_spec.rb

```diff
@@ -11,40 +11,23 @@
 
   it 'deposits objects' do
     ensure_token
-    job_id = SdrClient::Deposit.run(apo: APO,
-                                    source_id: source_id,
-                                    collection: COLLECTION,
-                                    catkey: catkey,
-                                    url: API_URL,
-                                    accession: true,
-                                    access: 'world',
-                                    files: ['Gemfile', 'Gemfile.lock'],
-                                    files_metadata: {
-                                      'Gemfile' => { 'preserve' => true },
-                                      'Gemfile.lock' => { 'preserve' => true }
-                                    })
-
-    # Wait for the deposit to be complete.
-    object_druid = nil
-    Timeout.timeout(Settings.timeouts.workflow) do
-      loop do
-        result = SdrClient::BackgroundJobResults.show(url: API_URL, job_id: job_id)
-        raise result[:output][:errors] if result[:output][:errors].present?
-
-        object_druid = result[:output][:druid]
-        break if object_druid
-      end
-    end
```

Comment:
I understand this is related to the bug you mentioned ... but don't we need to ensure it worked without error?  Or are we just waiting for 'v1 Accessioned` as sufficient?

Reply:
@ndushay This code just moved into `spec/support/deposit_helpers.rb`